tests/Tests/400__ods_dashboard/420__model_serving/420__model_serving.robot

```diff
@@ -0,0 +1,94 @@
+*** Settings ***
+Library           OperatingSystem
+Resource          ../../../Resources/Page/ODH/JupyterHub/HighAvailability.robot
+Suite Teardown    Teardown Model Serving
+
+
+*** Variables ***
+${MODEL_MESH_NAMESPACE}=    mesh-test
+${ODH_NAMESPACE}=    redhat-ods-applications
+${MS_REPO}=    https://github.com/opendatahub-io/modelmesh-serving
+${EXPECTED_INFERENCE_OUTPUT}=    {"model_name":"example-onnx-mnist__isvc-82e2bf7ea4","model_version":"1","outputs":[{"name":"Plus214_Output_0","datatype":"FP32","shape":[1,10],"data":[-8.233052,-7.749704,-3.4236808,12.363028,-12.079106,17.26659,-10.570972,0.7130786,3.3217115,1.3621225]}]}
+
+
+*** Test Cases ***
+Validate Model Serving quickstart
+    [Documentation]    Test the quickstart of the model serving repo, Temporary until included in RHODS
+    [Tags]    ModelMesh_Serving
+    # TODO: Replace with http://robotframework.org/robotframework/latest/libraries/Process.html#Run%20Process
+    Run    git clone ${MS_REPO}
+    Run    cd modelmesh-serving/quickstart && ./quickstart.sh ${ODH_NAMESPACE} ${MODEL_MESH_NAMESPACE}
```

Comment:
can we check a particular output string/code to assess if this is a PASS or FAIL?

Reply:
ok let's say it is more a suite setup maybe. Okay anyway